framework/include/st_things/st_things.h

```diff
@@ -152,6 +152,17 @@ int st_things_register_request_cb(st_things_get_request_cb get_cb, st_things_set
  */
 int st_things_start(void);
 
+/**
+ * @brief Send Representation of a specific resource through push service to Samsung Connect client.
+ * @param[in]  resource_uri Resource uri for which we want to send push message.
```

Comment:
It should be same as the name of the parameter. Pls change.

Reply:
fixed.